src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/Commands/MainListPage.cs

```diff
@@ -149,13 +150,13 @@ private bool ActuallyLoading()
     // _always_ show up first.
     private int ScoreTopLevelItem(string query, IListItem topLevelOrAppItem)
     {
-        if (string.IsNullOrWhiteSpace(query))
```

Comment:
Before:

For an empty **or whitespace** filter, we'd give everything a weight of 1...